lighthouse-core/test/report/html/renderer/report-ui-features-test.js

```diff
@@ -353,6 +353,22 @@ describe('ReportUIFeatures', () => {
         createDiv = () => dom.document().createElement('div');
       });
 
+      it('should return undefined when nodes is empty', () => {
+        const nodes = [];
+
+        const nextNode = dropDown._getNextSelectableNode(nodes);
+
+        assert.strictEqual(nextNode, undefined);
+      });
+
+      it('should return only node', () => {
```

Comment:
this test case feels identical to the next one to me, WDYT about covering the `_getNextSelectableNode([node], node)` case which feels a little more interesting?

```suggestion
      it('should return the only node when start is defined', () => {
```

Reply:
That sounds great to me, thanks for the suggestions! I will get an update out soon.

Reply:
Done!